src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java

```diff
@@ -20,6 +27,9 @@
     @PATCH("apis/{id}")
     Call<Api> updateApi(@Path("id") String nameOrId, @Body Api request);
 
+    /**
+     * @deprecated use {@link #createApi(Api)} or {@link #updateApi(String, Api)}
+     */
     @Deprecated
     @PUT("apis/")
     Call<Api> createOrUpdateApi(@Body Api request);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)